nservicebus/sagas/concurrency.md

```diff
@@ -151,6 +151,10 @@ In the above example, the requests are split into two groups each time. Dependin
 
 A simpler approach is to split the request just once and have a single level of sub-sagas sending the requests and aggregating the responses. This will not reduce data contention to the same degree, since the originating saga will be aggregating more multiple responses.
 
+#### Sequential scatter/gather
+
+Instead of sending out a large number of requests where simultanious processing of all responses is causing data contention consider sending the requests sequentially. Meaning, send the next request only after the response of the previous request has been processed. This will remove data contention in the procesing of responses but will increase the total duration the gathering of data will take and increase the saga state to store the data for all requests.
+
```

Comment:
```suggestion

Usually, the tradeoff is that the message that initiated the scatter/gather contains data that must now be stored in the saga state so that it can be added to each outgoing request in each iteration.

```